tools/lkl/tests/run_netperf.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# Usage
+#  ./run_netperf.sh [host_ip] [num_runs] [use_taskset] [test_name]
+
+set -e
+
+script_dir=$(cd $(dirname ${BASH_SOURCE:-$0}); pwd)
+hijack_script=${script_dir}/../bin/lkl-hijack.sh
+
+num_runs=1
+test_name="TCP_STREAM"
+use_taskset=0
+host_ip="192.168.13.1"
+taskset_cmd="taskset -c 1"
+test_len=10  # second
+
+if [ ! -x "$(which netperf)" ]; then
+    echo "WARNING: Cannot find a netserver executable, skipping netperf tests."
+    exit 0
+fi
+
+if [ $# -ge 1 ]
+then
+    host_ip=$1
+fi
+if [ $# -ge 2 ]
+then
+    num_runs=$2
+fi
+if [ $# -ge 3 ]
+then
+    use_taskset=$3
+fi
+if [ $# -ge 4 ]
+then
+    test_name=$4
+fi
+if [ $# -ge 5 ]
+then
+    echo "BAD NUMBER of INPUTS."
+    exit 1
+fi
+
+if [ $use_taskset -eq 0 ]
```

Comment:
Hmm, I guess it needs to be `if [[ -z $use_taskset ]]` or `if [ -z "$use_taskset" ]` ([from here](http://stackoverflow.com/questions/3427872/whats-the-difference-between-and-in-bash)). I thought that `-z` was the preferred way to do this but it seems your solution here and below [is the better one, anyway](http://stackoverflow.com/questions/3601515/how-to-check-if-a-variable-is-set-in-bash) so nevermind.